### PR TITLE
[refs #368] Add viewport height utilities

### PR DESCRIPTION
**Pull request by ajshortt**
https://github.com/inuitcss/inuitcss/issues/368

### CODE REVIEW

File: utilities/_utilities.viewport-heights.scss [hunk 1]
```diff
@@ -0,0 +1,129 @@
+/* ==========================================================================
+   #VIEWPORT-HEIGHTS
+   ========================================================================== */
+
+/**
+ * inuitcss generates a series of utility classes that give a height based on
+ * viewport height unit 'vh' to whichever element they’re applied, e.g.:
+ *
+ *   <div class="u-vh--100">
```
Comment (afontcu) [inline utilities/_utilities.viewport-heights.scss]: Hi! First of all, I really like the idea of having such utilities available by default, and I think that you came up with a quite nice solution :)

In this line you write utilities as modifiers, `u-vh--100`. However, everywhere else in the file you express these utilities as simple blocks (`u-vh-100`). I think that the former is what makes sense.

👋

File: utilities/_utilities.viewport-heights.scss [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+// for the classes with a `@` symbol so you get classes like:
+// <div class="u-vh-75@mobile">
+
+$inuit-viewports-breakpoint-separator: \@ !default;
+
+
+
+
+
+// A mixin to spit out our width classes. Pass in the columns we want the widths
```
Comment (afontcu) [inline utilities/_utilities.viewport-heights.scss]: I guess the comment should be updated :)

File: utilities/_utilities.viewport-heights.scss (hunk 1 above)
Comment (ajshortt) [reply utilities/_utilities.viewport-heights.scss]: Good spot. The modifier comment is actually a mistake. As `--` distinguishes a "modification" to a block in terms of BEM but these classes simply hold a single purpose. 

For example, `u-vh` is not generated as a block class so how can you modify that which does not exist 😆 .

**Comment by ajshortt** [comment]
Any news on this guys? @csshugs @csswizardry @herzinger

**Comment by florianbouvot** [comment]
@ajshortt I give you an [answer yesterday](https://github.com/inuitcss/inuitcss/issues/368#issuecomment-426017008)

**Pull request closed without merging**